group_vars/matrix_servers

```diff
@@ -1145,6 +1145,7 @@ matrix_coturn_enabled: true
 matrix_coturn_container_image_self_build: "{{ matrix_architecture != 'amd64' }}"
 
 matrix_coturn_turn_external_ip_address: "{{ ansible_host }}"
+matrix_coturn_turn_external_ip_addresses: [ "{{ matrix_coturn_turn_external_ip_address }}" ]
```

Comment:
This default should go (as-is) into `roles/matrix-coturn/defaults/main.yml`, because it applies not just to the playbook distribution, but also to the `matrix-coturn` role when used standalone (without the playbook overrides coming from `group_vars/matrix_servers`).

Reply:
Resolved